### PR TITLE
Explicitly ask for markdown in the response.

### PR DESCRIPTION
**Pull request by chhabrakadabra**

### CODE REVIEW

File: src/lib/rag.ts [hunk 1]
```diff
@@ -40,7 +40,13 @@ export class RagEngine {
             ["human", "{query}"],
         ]);
         const qaTemplate = ChatPromptTemplate.fromMessages([
-            ["system", "You are a helpful assistant that can answer questions about the user's notes. The user's notes are: \n{retrievedContext}"],
+            ["system", dedent`
+                You are a helpful assistant that can answer questions about the user's notes.
+                Use markdown to format your response.
```
Comment (hmitgang) [inline src/lib/rag.ts:45]: Should we use softer language like "You _may_ use markdown..." so that it only uses it where relevant?

Comment (chhabrakadabra) [reply src/lib/rag.ts:45]: I think it already does that without any explicit prompting. Also, when would it not be relevant? Additionally, we're already parsing the output as markdown.

Comment (chhabrakadabra) [reply src/lib/rag.ts:45]: I guess this is why projects have a large golden dataset to test against. We don't know what percentage of times md is appropriate vs not, so it just leads to differences in opinions on the team 😄 
I'm happy to switch to what you suggested. Maybe one day I'll build a test dataset and a tool to benchmark against it. That would be a good project.

Comment (hmitgang) [reply src/lib/rag.ts]: > I think it already does that without any explicit prompting.

Fair enough! I'm good with either :)

**Review by hmitgang — APPROVED** [review]

**Pull request merged**